content/types/_client_options.textile

```diff
@@ -83,3 +85,5 @@ blang[php].
   - logHandler := _@console.log@_ A function to handle each line of log output. If handler is not specified, @console.log@ is used. Note that the log level and log handler have global scope in the library and will therefore not act independently between library instances when multiple library instances exist concurrently.<br>__Type: @Function@__
 
 - <span lang="default">useBinaryProtocol</span><span lang="csharp">UseBinaryProtocol</span><span lang="python">use_binary_protocol</span><span lang="ruby">:use_binary_protocol</span> := <span lang="javascript">_false_ If set to true, will enable the binary protocol (MessagePack) if it is supported. It's disabled by default on browsers for performance considerations (browsers are optimized for decoding JSON)</span><span lang="default">_true_ If set to false, will forcibly disable the binary protocol (MessagePack). The binary protocol is used by default unless it is not supported</span><span lang="swift,objc,php">Note: The binary protocol is currently not supported </span><span lang="swift">in Swift</span><span lang="obj-c">in Objective-C</span><span lang="php">in PHP</span><span lang="javascript"></span>. Find out more about the "benefits of binary encoding":https://support.ably.io/solution/articles/3000047365-do-you-binary-encode-your-messages-for-greater-efficiency<br>__Type: @Boolean@__
+
+<!-- - <div lang="jsall,java,ruby"><span lang="default">logExceptionReportingUrl</span><span lang="ruby">:use_binary_protocol</span></div> := Defaults to a string value for an Ably error reporting Data Source Name. Typically of the form @https://[KEY]:[SECRET]@errors.ably.io/[ID]@. TODO: Once implemented, insert which values disable this.<br>__Type: @String@__ -->
```

Comment:
- `<span lang="ruby">:use_binary_protocol</span>`?
- Not sure why you've gated this to js, java, and ruby; none of those three implement this. AFAIK the only library that does is ably-ios. Gate to that (i.e. swift,objc) and have a look at the implementation of that to see what it needs to be set to to disable it

Reply:
From what I can tell from our spec sheet, it's not currently implemented for any libraries, but is meant to be enabled for ruby, java and js. If this is wrong happy to change it, I'll check our ios lib 
 now.

Reply:
OK but the user-facing documentation should reflect what's actually implemented, not what we might aspire to implement at some point

(btw I wouldn't read too much into the difference between `-` and `no` on the spreadsheet, in many cases it'll be more accident than a deliberate distinction between an item that should be implemented and one that shouldn't for that client lib)